### PR TITLE
temporal-cli: 1.16.2 -> 1.16.3

### PR DESCRIPTION
**Pull request by aaronjheng**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Mic92** [comment]
Result of `nixpkgs-review pr 194859` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>temporal-cli</li>
  </ul>
</details>

**Comment by Mic92** [comment]
This is what I get: 
```
--- PASS: TestUtilSuite (0.00s)
    --- PASS: TestUtilSuite/TestParseFoldStatusList (0.00s)
        --- PASS: TestUtilSuite/TestParseFoldStatusList/invalid (0.00s)
        --- PASS: TestUtilSuite/TestParseFoldStatusList/default_values (0.00s)
        --- PASS: TestUtilSuite/TestParseFoldStatusList/title_case (0.00s)
        --- PASS: TestUtilSuite/TestParseFoldStatusList/upper_case (0.00s)
        --- PASS: TestUtilSuite/TestParseFoldStatusList/no_values (0.00s)
    --- PASS: TestUtilSuite/TestParseKeyValuePairs (0.00s)
        --- PASS: TestUtilSuite/TestParseKeyValuePairs/no_key (0.00s)
        --- PASS: TestUtilSuite/TestParseKeyValuePairs/simple_values (0.00s)
        --- PASS: TestUtilSuite/TestParseKeyValuePairs/no_values (0.00s)
        --- PASS: TestUtilSuite/TestParseKeyValuePairs/empty (0.00s)
        --- PASS: TestUtilSuite/TestParseKeyValuePairs/no_separator (0.00s)
    --- PASS: TestUtilSuite/TestStringToEnum_MapCaseInsensitive (0.00s)
    --- PASS: TestUtilSuite/TestStringToEnum_MapEmptyEnum (0.00s)
    --- PASS: TestUtilSuite/TestStringToEnum_MapEmptyValue (0.00s)
    --- PASS: TestUtilSuite/TestStringToEnum_MapNonExisting (0.00s)
=== RUN   ExampleEnvProperty
--- FAIL: ExampleEnvProperty (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1111930]

goroutine 1 [running]:
testing.(*InternalExample).processRunResult(0xc00071fc68, {0xc0003a6040, 0x3a}, 0x1?, 0x0, {0x1243be0, 0x1ee8eb0})
        /nix/store/v4ys4lrjngf62lvvrdbs7r9kbxh9nqaa-go-1.18.6/share/go/src/testing/example.go:91 +0x4e5
testing.runExample.func2()
        /nix/store/v4ys4lrjngf62lvvrdbs7r9kbxh9nqaa-go-1.18.6/share/go/src/testing/run_example.go:59 +0x11c
panic({0x1243be0, 0x1ee8eb0})
        /nix/store/v4ys4lrjngf62lvvrdbs7r9kbxh9nqaa-go-1.18.6/share/go/src/runtime/panic.go:838 +0x207
github.com/temporalio/tctl/v2/cli.populateFlagsFunc.func1(0xc0005b7680)
        /build/source/cli/config.go:151 +0x270
github.com/urfave/cli/v2.(*App).RunAsSubcommand(0xc000526d00, 0xc0005b74c0)
        /build/source/vendor/github.com/urfave/cli/v2/app.go:421 +0x89b
github.com/urfave/cli/v2.(*Command).startApp(0x1efc1e0, 0xc0005b74c0)
        /build/source/vendor/github.com/urfave/cli/v2/command.go:280 +0x76f
github.com/urfave/cli/v2.(*Command).Run(0xc000239180?, 0xc000239180?)
        /build/source/vendor/github.com/urfave/cli/v2/command.go:94 +0xba
github.com/urfave/cli/v2.(*App).RunContext(0xc0005269c0, {0x15eada0?, 0xc000044258}, {0xc0005b6f00, 0x4, 0x4})
        /build/source/vendor/github.com/urfave/cli/v2/app.go:313 +0xb48
github.com/urfave/cli/v2.(*App).Run(...)
        /build/source/vendor/github.com/urfave/cli/v2/app.go:224
github.com/temporalio/tctl/v2/cli.setupConfig(0xc0005269c0)
        /build/source/cli/config_test.go:66 +0x9d
github.com/temporalio/tctl/v2/cli.ExampleEnvProperty()
        /build/source/cli/config_test.go:37 +0x38
testing.runExample({{0x13e17f2, 0x12}, 0x146e150, {0x144c666, 0x84}, 0x0})
        /nix/store/v4ys4lrjngf62lvvrdbs7r9kbxh9nqaa-go-1.18.6/share/go/src/testing/run_example.go:63 +0x28d
testing.runExamples(0xc00071fe58, {0x1ef9380?, 0x3, 0x4?})
        /nix/store/v4ys4lrjngf62lvvrdbs7r9kbxh9nqaa-go-1.18.6/share/go/src/testing/example.go:44 +0x186
testing.(*M).Run(0xc000232140)
        /nix/store/v4ys4lrjngf62lvvrdbs7r9kbxh9nqaa-go-1.18.6/share/go/src/testing/testing.go:1721 +0x689
main.main()
        _testmain.go:59 +0x1aa
FAIL    github.com/temporalio/tctl/v2/cli       0.033s
FAIL
```

File: pkgs/applications/networking/cluster/temporal-cli/default.nix [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
   };
 
-  vendorSha256 = "sha256-kczmoP32/V0HHeC3Mr+giuMB+McVTNeC2F+t1ohY4/U=";
+  vendorSha256 = "sha256-9bgovXVj+qddfDSI4DTaNYH4H8Uc4DZqeVYG5TWXTNw=";
 
   ldflags = [ "-s" "-w" ];
 
-  passthru.tests.version = testers.testVersion {
-    package = temporal-cli;
-  };
```
Comment (Mic92) [inline pkgs/applications/networking/cluster/temporal-cli/default.nix:20]: Why was this removed?

**Comment by aaronjheng** [comment]
Closing in favor of #197810
